tests/io/feeding_services/http_tests.py

```diff
@@ -70,12 +69,19 @@ def test_get_expired_token(self):
         self.assertEquals('', TestFeedingService()._get_auth_token(provider))
 
     def test_fetch_token(self):
+        # TODO: need some rewriting
```

Comment:
but the test name is `test_fetch_token`, so we need some replacement for that scenario, this why I'm keeping it there